packages/build-utils/src/fs/run-user-scripts.ts

```diff
@@ -141,6 +156,18 @@ export async function getNodeBinPath({
   return path.join(dir, 'node_modules', '.bin');
 }
 
+export function getNodeBinPaths({
+  start,
+  root,
```

Comment:
Can we change `root` to `base` in these new functions to match the existing ones?

It also matches the option from `@vercel/nft` https://github.com/vercel/nft#options

Reply:
29c0e0adb542b2ae793dee598a1aec9e6a1627db